.github/workflows/docker-tests.yml

```
@@ -42,92 +66,84 @@ jobs:
           # Create a JSON file for Testing Farm in order to schedule a CI testing job
```

```suggestion
          # Create a JSON data structure for GitHub API to display the test status in the PR
```
I am not sure if I understand this correctly, but isn't this request pointing to the GitHub API, and not to to Testing Farm?